src/core/hle/service/nwm/nwm_uds.h

```diff
@@ -73,9 +73,15 @@ enum class NetworkStatus {
     ConnectedAsSpectator = 10,
 };
 
+enum class NetworkStatusChangeReason {
+    None = 0,
+    ConnectionEstablished = 1,
+    ConnectionLost = 4,
+};
+
 struct ConnectionStatus {
     u32_le status;
-    INSERT_PADDING_WORDS(1);
+    u32_le status_change_reason;
```

Comment:
```suggestion
    enum_le<NetworkStatusChangeReason> status_change_reason;
```
Would it be reasonable to store this as the enum instead of an integer?